command/ca/provisioner/update.go

```diff
@@ -609,6 +612,17 @@ func updateACMEDetails(ctx *cli.Context, p *linkedca.Provisioner) error {
 	if ctx.IsSet("challenge") || ctx.IsSet("remove-challenge") {
 		details.Challenges = sliceutil.RemoveDuplicates(details.Challenges)
 	}
+	if ctx.IsSet("remove-attestation-format") {
```

Comment:
Shouldn't we be validating these? like we do for provisioner add? Same question about the `challenge` flag.

Reply:
Fixed with 021d372